README.md

```diff
@@ -1,24 +1,21 @@
-# ![nf-core/methylseq](docs/images/methylseq_logo.png)
+# ![nf-core/methylseq](docs/images/nf-core-methylseq_logo.png)
 
-[![Build Status](https://travis-ci.org/nf-core/methylseq.svg?branch=master)](https://travis-ci.org/nf-core/methylseq)
-[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.32.0-brightgreen.svg)](https://www.nextflow.io/)
+**Methylation (Bisulfite-Sequencing) Best Practice analysis pipeline, part of the nf-core community.**.
+
+[![Build Status](https://travis-ci.com/nf-core/methylseq.svg?branch=master)](https://travis-ci.com/nf-core/methylseq)
+[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A519.04.0-brightgreen.svg)](https://www.nextflow.io/)
 [![DOI](https://zenodo.org/badge/124913037.svg)](https://zenodo.org/badge/latestdoi/124913037)
 [![Gitter](https://img.shields.io/badge/gitter-%20join%20chat%20%E2%86%92-4fb99a.svg)](https://gitter.im/nf-core/Lobby)
```

Comment:
That should not be there.
We're on slack now ;-)

Reply:
Good spot! We should probably lint for this..